mkdocs.yml

```diff
@@ -14,6 +14,7 @@ nav:
 - Overview: index.md
 - Trestle cli: cli.md
 - Tutorials:
+  - intro to trestle workflow: tutorials/trestle_sample_workflow.md
   - transformer construction: tutorials/task.transformer-construction/transformer-construction.md
```

Comment:
Missing inclusion of the python trestle setup.

Reply:
added